src/Map.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import GoogleMapReact from 'google-map-react';
+
+//TODO: we will need to create markers for each post type. Markers
+//      are just divs that are rendered at specific lat longs on
+//      the map. 
+/*const simpleMarker = ({ text }) => (
+    <div style={{
+      color: 'white', 
+      background: 'grey',
+      padding: '15px 10px',
+      display: 'inline-flex',
+      textAlign: 'center',
+      alignItems: 'center',
+      justifyContent: 'center',
+      borderRadius: '100%',
+      transform: 'translate(-50%, -50%)'
+    }}>
+      {text}
+    </div>
+  );
+*/
+
+class CSMap extends Component {
```

Comment:
Might want to change this to a functional component and use hooks for state management.